lib/jnpr/junos/device.py

```diff
@@ -279,6 +279,40 @@ def master(self, value):
         raise RuntimeError("master is read-only!")
 
     # ------------------------------------------------------------------------
+    # property: uptime
+    # ------------------------------------------------------------------------
+
+    @property
+    def uptime(self):
+        """
+        The uptime of the current Routing Engine.
+
+        The current Routing Engine is the RE to which the NETCONF session is
+        connected.
+
+        :returns: The number of seconds (int) since the current Routing Engine
+                  was booted. If there is a problem gathering or parsing the
+                  uptime information, None is returned.
+        :raises: May raise a specific jnpr.junos.RpcError or
+                 jnpr.junos.ConnectError subclass if there is a problem
+                 communicating with the device.
+        """
+        uptime = None
```

Comment:
If for some reason the expected XML element doesn't exist, I think it makes more sense to return `None` than `0`.  This is also documented in the property:
```
If there is a problem gathering or parsing the  uptime information, None is returned.
```

The `None` value is also consistent with the `master` property and the facts in `dev.facts`. All of those return `None` if the information is unable to be gathered from the device.